integration_tests/ali/ali_test.py

```diff
@@ -7,13 +7,15 @@
 def test_ali_instances(ali_client):
     region_id = ali_client.get_region_id()
     assert(region_id == 'eu-central-1')
-    
+
     request = DescribeInstancesRequest.DescribeInstancesRequest()
     request.set_PageSize(10)
 
     response = ali_client.do_action_with_exception(request)
     instances_dict = json.loads(response.decode())
     assert(len(instances_dict) > 0)
-    inst = instances_dict['Instances']['Instance'][0]
-    assert(inst)
-    assert(inst['RegionId'] == region_id)
\ No newline at end of file
+    if 'Instances' in instances_dict and 'Instance' in instances_dict['Instances'] \
+        and len(instances_dict['Instances']['Instance']) > 0:
+        inst = instances_dict['Instances']['Instance'][0]
+        assert(inst)
```

Comment:
nit: no need for parenthesis around `assert`s

Reply:
👍 so true